asterisk/Dockerfile

```diff
@@ -254,6 +258,21 @@ RUN ./bootstrap; \
     make install; \
     cp -f etc/dongle.conf "${INSTALL_DIR}/etc/asterisk/"
 
+## Install chan-quectel
+WORKDIR /usr/src/asterisk-chan-quectel
+# https://github.com/RoEdAl/asterisk-chan-quectel/commits/master
+ARG CHAN_QUECTEL_VERSION="5552c365bfb319eed7cbbf6300a67028ab70db9e"
+RUN curl -fsSL "https://github.com/RoEdAl/asterisk-chan-quectel/archive/${CHAN_QUECTEL_VERSION}.tar.gz" | \
```

Comment:
@litinoveweedle why not https://github.com/IchthysMaranatha/asterisk-chan-quectel?

https://github.com/RoEdAl/asterisk-chan-quectel is archived, doesn't look like a good idea.